system76driver/products.py

```diff
@@ -160,6 +160,12 @@
             actions.energystar_wakeonlan,
 	    ],
     },
+    'galp3-c': {
+        'name': 'Galago Pro',
+        'drivers': [
+            actions.internal_mic_gain,
```

Comment:
We should leave this empty. It may not need internal mic gain fix

Reply:
I had kinda left it in as a reminder to test this, but that makes sense to default empty.